benchmarks/run.js

```diff
@@ -1,17 +1,20 @@
-var fixtures = require('./fixtures');
-var local = require('../');
-var dedupe = require('../dedupe');
-var localPackage = require('../package.json');
+import { readFile } from 'fs/promises'
+import fixtures from './fixtures.js';
+import local from '../index.js';
+import dedupe from '../dedupe.js';
+
+var localPackage = JSON.parse(await readFile(new URL('../package.json', import.meta.url)));
 
 function log (message) {
 	console.log(message);
 }
 
 try {
-	var npm = require('classnames');
-	var npmDedupe = require('classnames/dedupe');
-	var npmPackage = require('./node_modules/classnames/package.json');
+	var npm = (await import('classnames')).default;
+	var npmDedupe = (await import('classnames/dedupe.js')).default;
+	var npmPackage = JSON.parse(await readFile(new URL('./node_modules/classnames/package.json', import.meta.url)));
```

Comment:
These top-level awaits are only supported from Node 14 and up, so we'll need a fix for older versions.